yt/default.mplstyle

```diff
@@ -0,0 +1,15 @@
+# basic usage (requires matplotlib 3.7+)
+# >>> import matplotlib as mpl
+# >>> mpl.style.use("yt.default")
+
+xtick.top: True
+ytick.right: True
+xtick.minor.visible: True
+ytick.minor.visible: True
+xtick.direction: in
+ytick.direction: in
+
+font.family: stixgeneral
+
+mathtext.fontset: cm
+mathtext.fallback: cm
```

Comment:
hint to reviewers: this option is the reason why I'm dropping support for matplotlib 3.2 in the same PR (it's invalid before matplotlib 3.3). Previously this was handled dynamically but obviously this behaviour cannot be replicated with a static style sheet.